src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
```

Comment:
type Shapes = 'triangle' | 'circle' | 'rectangle';
type Colors = 'red' | 'green' | 'blue';
shape: Shapes;
color: Colors;

if you define enumerations, you won’t have to give types every time in Class, only change a shape type